src/generators/azuredevops/index.ts

```diff
@@ -151,10 +152,16 @@ class Main extends Generator {
         tenantId: this.answers.tenantId,
       });
       this.log('Done.');
-    } catch (e) {
-      this.log('Package generator encountered an error.');
-      await scaffolder.rollback(this.answers.adoProject);
-      this.log(e as string);
+    } catch (scaffoldError) {
+      this.log(chalk.red('Package generator encountered an error.'));
+      try {
+        await scaffolder.rollback(this.answers.adoProject);
+      } catch (rollbackError) {
+        this.log(chalk.red('Rollback failed: '));
+        this.log(chalk.red(rollbackError as string));
+        this.log(chalk.red('--------------------------------------'));
+      }
+      this.log(chalk.red(scaffoldError as string));
```

Comment:
@tdashworth - can you move up the line 164, scaffold error log to, after line 156. Also it would be good to add a prefix string for both scaffold error log and rollback error log(line 161) to clearly distinguish the error was due to scaffold or rollback.

Reply:
I've reordered the error messages but I've skipped the prefixing as I believe the separate logs should be enough.